### DISCUSSION TITLE
Heic supoort.

### DISCUSSION THREAD

**Discussion by incrediblezayed** [rocksdanister/lively#258] [Ideas]
Please add support for heic files. There are tons of dynamic wallpapers available with heic extensions, it'll be awesome if we have it.

**Comment by rocksdanister** [comment]
The Mac dynamic wallpaper file? Because Heic just looks like multiple images in a single file, the dynamic wallpaper file is some implementation based on that.

**Reply by incrediblezayed** [reply]
Yes, the heic are multiple images, but displaying them over time is good.

**Reply by rocksdanister** [reply]
Hmm .. I don't know.. there are other projects specialized on this task:
https://github.com/t1m0thyj/WinDynamicDesktop

Lively is mainly a live wallpaper app.
